esp-hal/src/clock/mod.rs

```diff
@@ -591,65 +599,136 @@ impl Clocks {
     }
 }
 
-/// Control the radio peripheral clocks
 #[cfg(any(bt, ieee802154, wifi))]
+/// Tracks which modems currently request an active PHY clock.
```

Comment:
Technically this isn't true anymore, this just track how many users keep the PHY clock alive :)